application/frontend/config/main.php

```diff
@@ -30,14 +33,18 @@
             // http://www.yiiframework.com/doc-2.0/guide-rest-routing.html
             'rules' => [
                 [
+                    // http://www.yiiframework.com/doc-2.0/yii-rest-urlrule.html
                     'class' => 'yii\rest\UrlRule',
-                    'controller' => ['user', 'authentication'],
+                    'controller' => ['authentication', 'user'],
+                    'extraPatterns' => [
```

Comment:
I would have simply listed all half-dozen of the endpoints specifically, like the example on (old) line 38. That may just be leftover practices from my Yii1 tendencies, though. Feel free to go with whatever you prefer here.